include/caffe/util/cudnn.hpp

```diff
@@ -8,7 +8,7 @@
 #include "caffe/proto/caffe.pb.h"
 
 #define CUDNN_VERSION_MIN(major, minor, patch) \
-    (CUDNN_VERSION >= (major * 1000 + minor * 100 + patch))
+     (CUDNN_VERSION >= (major * 1000 + minor * 100 + patch))
```

Comment:
I assume this whitespace change is an error and should not be here? (If there is some reason for it, please do not put it in the same commit as unrelated code.)